### PR TITLE
Legend component V9

### PR DESCRIPTION
**Pull request by srmukher**
Legend component V9
image

### CODE REVIEW

File: packages/react-components/react-charts-preview/src/components/Legends/Legends.tsx [hunk 1]
```diff
@@ -1,25 +1,13 @@
 import * as React from 'react';
 
-import { IContextualMenuItem } from '@fluentui/react/lib/ContextualMenu';
-import { HoverCard, HoverCardType, IExpandingCardProps } from '@fluentui/react/lib/HoverCard';
-import { classNamesFunction, find, getNativeProps, buttonProperties } from '@fluentui/react/lib/Utilities';
-import { ResizeGroup } from '@fluentui/react/lib/ResizeGroup';
-import { IProcessedStyleSet } from '@fluentui/react/lib/Styling';
-import { OverflowSet, IOverflowSetItemProps } from '@fluentui/react/lib/OverflowSet';
+import { getNativeProps, buttonProperties } from '@fluentui/react/lib/Utilities';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/Legends/Legends.tsx:3]: dont use any v8 components

File: packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+} from '../../../../react-components/src/index';
+
+export const OverflowMenu: React.FC<{ itemIds: string[]; title: string; items: JSX.Element[] }> = ({
+  itemIds,
+  title,
+  items,
+}) => {
+  const { ref, overflowCount, isOverflowing } = useOverflowMenu<HTMLButtonElement>();
+  let displayLabel = title;
+  displayLabel = title === '' ? `+${overflowCount} items` : `+${overflowCount} ${title}`;
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx:19]: have you tested this control on RTL

File: packages/react-components/react-charts-preview/stories/Legends/LegendsDefault.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+      },
+    },
+  ];
+  return <Legends legends={legends} />;
+};
+
+LegendsBasic.parameters = {
+  docs: {
+    description: {
+      story: 'Donut Chart Story.',
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/stories/Legends/LegendsDefault.stories.tsx:178]: correct this

File: packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx (hunk 2 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx:19]: yes, its working in RTL
image

File: packages/react-components/react-charts-preview/stories/Legends/LegendsDefault.stories.tsx (hunk 3 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/stories/Legends/LegendsDefault.stories.tsx:178]: done

File: packages/react-components/react-charts-preview/stories/Legends/LegendsDefault.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,181 @@
+import * as React from 'react';
```
[the remark this answers is not in the document]
Comment (srmukher) [reply packages/react-components/react-charts-preview/stories/Legends/LegendsDefault.stories.tsx:1]: added

File: packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  useOverflowMenu,
+} from '../../../../react-components/src/index';
```
[the remark this answers is not in the document]
Comment (srmukher) [reply packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx:10]: done

File: packages/react-components/react-charts-preview/src/components/Legends/Legends.tsx (hunk 1 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/src/components/Legends/Legends.tsx:3]: updated with getIntrinsicElementProps

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-components/react-charts-preview/stories/Legends/Legends.Overflow.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+      allowFocusOnLegends={true}
+      canSelectMultipleLegends={false}
+    />
+  );
+};
+
+LegendsOverflow.parameters = {
+  docs: {
+    description: {
+      story: 'Donut Chart Story.',
```
[the remark this answers is not in the document]
Comment (srmukher) [reply packages/react-components/react-charts-preview/stories/Legends/Legends.Overflow.stories.tsx]: done for all

**Pull request merged**